uninstall.php

```diff
@@ -24,3 +24,39 @@
 delete_option( 'jetpack_was_activated'  );
 delete_option( 'jetpack_auto_installed' );
 delete_transient( 'jetpack_register'    );
+
+// Jetpack Sync
+/**
+ * Remove the sync queue
+ */
+function jetpack_remove_sync_queue() {
+	global $wpdb;
+	$wpdb->query( $wpdb->prepare(
+		"DELETE FROM $wpdb->options WHERE option_name LIKE %s", "jpsq_sync-%"
+	);
+}
+
+/**
+ * Remove the default sync settings
+ */
+function jetpack_remove_sync_settings() {
+	$valid_settings = array( 'dequeue_max_bytes' => true, 'upload_max_bytes' => true, 'upload_max_rows' => true, 'sync_wait_time' => true );
```

Comment:
Just in case these change, you probably want to use the Jetpack_Sync_Client "$valid_settings" property, perhaps wrapped in a method, like:

```php
$valid_settings = Jetpack_Sync_Queue::getInstance()->get_valid_settings();
```